azure-pipelines.yml

```diff
@@ -113,42 +113,75 @@ stages:
+              SlotName: production
+              packageForLinux: '$(System.DefaultWorkingDirectory)/build_linux_$(Build.BuildId).zip'
           - task: AzureRmWebAppDeployment@4
-            displayName: "Deploy Windows Functions"
+            condition: eq(variables['TestPublishing'], 'true')
+            displayName: "Deploy Windows Functions to Test"
             inputs:
               ConnectionType: 'AzureRM'
               # Name of the service connection that will give us access to Dotnet-Install-Monitoring resource group
               # This is where the windows based Azure Functions are allocated.
               azureSubscription: 'InstallScripts_KeyVault'
               appType: 'functionApp'
-              WebAppName: 'dotnet-install-daemon-functions'
+              WebAppName: dotnet-install-daemon-functions
               deployToSlotOrASE: true
               ResourceGroupName: 'Dotnet-Install-Monitoring'
-              SlotName: 'production'
+              SlotName: test
               packageForLinux: '$(System.DefaultWorkingDirectory)/build_windows_$(Build.BuildId).zip'
+
           - task: AzureRmWebAppDeployment@4
-            displayName: "Deploy Linux Functions"
+            condition: eq(variables['TestPublishing'], 'false')
+            displayName: "Deploy Linux Functions to Prod"
             inputs:
               ConnectionType: 'AzureRM'
               # Name of the service connection that will give us access to Dotnet-Install-Monitoring-Visualization resource group
               # This is where the linux based Azure Functions are allocated.
               azureSubscription: 'InstallScripts_Visualization_RG'
               appType: 'functionApp'
-              WebAppName: 'dotnet-install-daemon-functions-linux'
+              WebAppName: dotnet-install-daemon-functions-linux
               deployToSlotOrASE: true
               ResourceGroupName: 'Dotnet-Install-Monitoring-Visualization'
-              SlotName: 'production'
+              SlotName: production
               packageForLinux: '$(System.DefaultWorkingDirectory)/build_linux_$(Build.BuildId).zip'
+          - task: AzureRmWebAppDeployment@4
+            condition: eq(variables['TestPublishing'], 'false')
+            displayName: "Deploy Windows Functions to Prod"
+            inputs:
+              ConnectionType: 'AzureRM'
+              # Name of the service connection that will give us access to Dotnet-Install-Monitoring resource group
+              # This is where the windows based Azure Functions are allocated.
+              azureSubscription: 'InstallScripts_KeyVault'
+              appType: 'functionApp'
+              WebAppName: dotnet-install-daemon-functions
+              deployToSlotOrASE: true
+              ResourceGroupName: 'Dotnet-Install-Monitoring'
+              SlotName: production
+              packageForLinux: '$(System.DefaultWorkingDirectory)/build_windows_$(Build.BuildId).zip'
```

Comment:
This line refers to both Linux and Windows. Is it expected?

Reply:
`packageForLinux` is just an alias for `Package`, but I agree that it looks wrong.
I replaced it with `Package` now.